content/en/docs/Examples/localai.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "LocalAI"
+linkTitle: "LocalAI"
+weight: 4
+description: > 
+    This section describe examples on how to deploy Kairos with k3s and LocalAI
+---
+
+{{% alert title="Warning" %}}
+This tutorial will download an AI model which is around 4Gib
```

Comment:
I'll add a note that performance are poors on VMs, just bare metal with modern HW.